sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -474,6 +486,15 @@ private static DefaultNetworkSpec from(
         return ports;
     }
 
+    private static boolean maybeUsePortResources(Collection<String> networkNames) {
```

Comment:
Nit pick: Maybe sounds like it's going to return an optional.

I'd change it to `willUsePortResources`